src/abaqus/Adaptivity/VelocityAdaptiveMeshConstraint.py

```diff
@@ -25,31 +27,31 @@ class VelocityAdaptiveMeshConstraint(AdaptiveMeshConstraint):
 
     #: A SymbolicConstant specifying the category of the adaptive mesh constraint. Possible
     #: values are MECHANICAL and THERMAL.
-    category: Optional[SymbolicConstant] = None
+    category: Optional[Literal[MECHANICAL, THERMAL]] = None
 
     #: A :py:class:`~abaqus.Region.Region.Region` object specifying the region to which the adaptive mesh constraint is applied.
     region: Region = Region()
 
     #: None or a DatumCsys object specifying the local coordinate system of the adaptive mesh
     #: constraint's degrees of freedom. If **localCsys** = None, the degrees of freedom are defined
     #: in the global coordinate system. The default value is None.
-    localCsys: Optional[str] = None
+    localCsys: Optional[DatumCsys] = None
 
     @abaqus_method_doc
     def __init__(
         self,
         name: str,
         createStepName: str,
         region: Region,
-        v1: Union[SymbolicConstant, float] = UNSET,
-        v2: Union[SymbolicConstant, float] = UNSET,
-        v3: Union[SymbolicConstant, float] = UNSET,
-        vr1: Union[SymbolicConstant, float] = UNSET,
-        vr2: Union[SymbolicConstant, float] = UNSET,
-        vr3: Union[SymbolicConstant, float] = UNSET,
-        amplitude: str = UNSET,
-        localCsys: Optional[str] = None,
-        motionType: SymbolicConstant = INDEPENDENT,
+        v1: Union[float, Literal[SET,UNSET]] = UNSET,
+        v2: Union[float, Literal[SET,UNSET]] = UNSET,
+        v3: Union[float, Literal[SET,UNSET]] = UNSET,
+        vr1: Union[float, Literal[SET,UNSET]] = UNSET,
+        vr2: Union[float, Literal[SET,UNSET]] = UNSET,
+        vr3: Union[float, Literal[SET,UNSET]] = UNSET,
```

Comment:
Can you add spaces after the commas?

Reply:
Ok!

Reply:
Also, there are other files where to remove `None`

Reply:
We may correct those later on, there are a lot of things to do.